### DISCUSSION TITLE
Is there a feature to save conversations as notes?

### DISCUSSION THREAD

**Discussion by awwsome** [glowingjade/obsidian-smart-composer#71] [Ideas & Feature Requests]
안녕하세요. 플러그인 설치해서 잘 사용하고 있습니다.
대화를 나눈 내용을 바로 노트에 저장할 수 있을까요? 기능이 있는데 제가 못 찾는 것인지, 아직 제공되고 있지 않은 것인지 모르겠네요.
감사합니다.

(혹시 게시물을 영어로 작성해야 한다면 말씀해 주세요)

---
**(English Translation)**

Hello. I've installed the plugin and I'm using it well.
Is it possible to save the conversation directly to notes? I'm not sure if this feature exists and I'm just unable to find it, or if it's not yet available.
Thank you.

(Please let me know if the post should be written in English)

**Comment by glowingjade** [comment]
@awwsome 
안녕하세요.  
대화 내용을 노트에 저장하고 싶다는 것은 AI와의 대화 내역 전체를 저장하고 싶다는 말씀인가요? 아니면 AI가 제안한 수정 사항만을 노트에 반영하고 싶다는 의미인가요?
만약 후자라면, Apply 버튼을 눌러서 제안된 수정 사항을 반영할 수 있습니다!
[참고](https://github.com/glowingjade/obsidian-smart-composer?tab=readme-ov-file#apply-edit)

또한 게시물은 한국어로 작성해 주셔도 되며, 게시글을 저희가 직접 수정하여 영어로 번역해서 병기할 예정입니다.

---
**(English Translation)**

Hello.
Are you saying you want to save the entire conversation history with the AI to your notes? Or do you mean you only want to reflect the modifications suggested by the AI in your notes?
If it's the latter, you can apply the suggested modifications by pressing the Apply button!
[Reference](https://github.com/glowingjade/obsidian-smart-composer?tab=readme-ov-file#apply-edit)

Also, you're welcome to write your posts in Korean, and we will directly edit and translate them into English to provide a bilingual version.

**Reply by awwsome** [reply]
답변 감사합니다. 저는 전자의 경우를 여쭤봤습니다 :)
말씀하신 것처럼 기존 노트를 수정하는 상황도 있겠지만, 기존 노트들을 조합해 새로운 글을 작성하게 하는 상황도 많을 것 같은데, 이를 위해 AI가 응답한 메시지 전체를 마크다운으로 저장할 수 있으면 편할 것 같습니다.
유사한 플러그인 중 '[obsidian-copliot](https://github.com/logancyang/obsidian-copilot)' 이라는 제품에 Save as Note 버튼이 있습니다.

---
**(English Translation)**

Thank you for your response. I was asking about the former case :)
As you mentioned, there may be situations where existing notes are modified, but I think there will be many situations where new content is created by combining existing notes. For this purpose, it would be convenient if the entire message responded by AI could be saved as markdown.
Among similar plugins, there's a product called '[obsidian-copilot](https://github.com/logancyang/obsidian-copilot)' which has a 'Save as Note' button.

**Reply by glowingjade** [reply]
아하 이해했습니다!
저희 로드맵에 추가해두었습니다:
https://github.com/users/glowingjade/projects/1/views/3?pane=issue&itemId=85651515
빠른 시일 내에 지원할 수 있도록 해보겠습니다 :)
감사합니다.

---
**(English Translation)**

Oh, I understand!
We've added it to our roadmap:
https://github.com/users/glowingjade/projects/1/views/3?pane=issue&itemId=85651515
We'll try to support it as soon as possible :)
Thank you.

**Reply by HomeHeartTherapy** [reply]
And it would be great to have a checkbox (or something) to preserve the markdown formatting :)  Thanks!

**Comment by BradKML** [comment]
This feels like what Cursor should be doing already, but for notes https://github.com/somogyijanos/cursor-chat-export

**Comment by hiitssai** [comment]
I'd love to see this feature implemented, I'm happy to contribute to its completion if that's welcome.

**Comment by sonct988** [comment]
It's been over a year and this feature still hasn't been implemented :(. I hope you'll get it done soon. Smart Composer is only inferior to Obsidian Copilot in this feature. I've really been waiting a long time.

**Comment by HomeHeartTherapy** [comment]
I just started trying to code a pop-up modal that allows the users to search, scroll through chats, etc and export.  I'll let you know if I make any real headway.  Of course, I'll wire it to a meta bind and maybe through that into Note Toolbar (or something).

**Comment by sonct988** [comment]
![CleanShot 2025-11-25 at 12 13 26@2x](https://github.com/user-attachments/assets/c97e1568-6f7a-47d0-b807-41f187d49eb2)

I hope it has similar functionality to the copilot plugin. Copilot is good, but I still prefer how smart composer applies changes.
